PyPDF2/_reader.py

```diff
@@ -733,7 +731,7 @@ def getOutlines(
             PendingDeprecationWarning,
             stacklevel=2,
         )
-        return self.get_outlines(node, outlines)
+        return self._get_outlines(node, outlines)
```

Comment:
You will want to update the docstring and warning above to reference the `outlines` property instead of the non-existent `get_outlines` method.

Reply:
Good catch! Thank you for having a look :hugs: